extensions/webirc.md

```diff
@@ -44,6 +56,14 @@ IP address does not resolve to hostname.
 
     WEBIRC hunter2 ExampleGateway 198.51.100.3 198.51.100.3
 
+Secure connection.
+
+    WEBIRC hunter2 ExampleGateway 198.51.100.3 198.51.100.3 secure
+
+Secure connection with ports passed through.
+
+    WEBIRC hunter2 ExampleGateway 198.51.100.3 198.51.100.3 :secure=examplevalue local-port=6697 remote-port=21726
```

Comment:
You didn't say secure could take a value

Reply:
[This line](https://github.com/ircv3/ircv3-specifications/pull/328/files#diff-407b2883275563cf9468d4506dae17ceR44) indicates that servers MUST be able to handle keys that don't have values right now gaining values in the future. This example explicitly includes a value to make sure servers don't trip over and can comply with that statement.